.github/workflows/create-jira-refactor-issue.yml

```
@@ -0,0 +1,89 @@
+name: Create Refactor issue in Jira
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  create-issue:
+    if : "startsWith(github.event.issue.title, 'Refactor')"
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)